apps/charterafrica/src/lib/openAfrica.js

```diff
@@ -123,3 +123,24 @@ export default async function fetchDatasets(organization, query = {}) {
     return error;
   }
 }
+
+export async function fetchDataset(id) {
+  try {
+    const response = await fetchJson.get(
+      `${BASE_DOCUMENTS_URL}package_show?id=${id}`
+    );
+    const { result: dataset } = response;
+    const { tags = [] } = dataset;
+    const tagsNames = tags.map((tag) => tag.name);
```

Comment:
Have we confirmed that all datasets have tags? Otherwise we may have to have a backup plan e.g. tags first if available, groups if no tags, etc.